src/main/java/core/basesyntax/db/DaoService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.db;
+
+import java.util.List;
+
+public interface DaoService {

```

Comment:
```suggestion
public interface FruitDao {
```
I'd rather recommend naming it this way. The same for implementation